### PR TITLE
Logging: datetime and new log entry

### PR DESCRIPTION
**Pull request by aarani**

### CODE REVIEW

File: NOnion/TorLogger.fs [hunk 1]
```diff
@@ -9,6 +9,8 @@ module TorLogger =
         LoggerOpt <- Some loggingFunc
 
     let Log(msg: string) =
+        let msgWithDateTime = sprintf "[%s] %s" (DateTime.Now.ToString()) msg
+
         match LoggerOpt with
         | None -> ()
-        | Some logger -> logger.Invoke(msg)
```
Comment (knocte) [inline NOnion/TorLogger.fs:14]: no need for parenthesis here

File: NOnion/TorLogger.fs [hunk 2]
```diff
@@ -9,6 +9,8 @@ module TorLogger =
...
+        | Some logger -> logger.Invoke(msgWithDateTime)
```
Comment (knocte) [inline NOnion/TorLogger.fs:16]: no need for parenthesis here

File: NOnion/TorLogger.fs [hunk 3]
```diff
@@ -9,6 +9,8 @@ module TorLogger =
         LoggerOpt <- Some loggingFunc
 
     let Log(msg: string) =
+        let msgWithDateTime = sprintf "[%s] %s" (DateTime.Now.ToString()) msg
```
Comment (knocte) [inline NOnion/TorLogger.fs:12]: should it be UTC?

**Comment by knocte** [comment]
`NOnion:` is not valid as area of code.

**Comment by aarani** [comment]
> `NOnion:` is not valid as area of code.

It's in the root namespace.

File: NOnion/TorLogger.fs (hunk 3 above)
Comment (aarani) [reply NOnion/TorLogger.fs:12]: I personally prefer this, but both are fine to me

**Comment by knocte** [comment]
> It's in the root namespace.

So what? The area doesn't map to namespaces. This bit of information needs to be useful and using "NOnion: " doesn't make it useful because the repo/lib is called like that already.

File: NOnion/TorLogger.fs (hunk 3 above)
Comment (aarani) [reply NOnion/TorLogger.fs]: Changed to UTC

File: NOnion/TorLogger.fs (hunk 2 above)
Comment (aarani) [reply NOnion/TorLogger.fs]: Done

**Comment by aarani** [comment]
> > It's in the root namespace.
> 
> So what? The area doesn't map to namespaces. This bit of information needs to be useful and using "NOnion: " doesn't make it useful because the repo/lib is called like that already.

Changed to "TorLogger"

**Pull request merged**